### PR TITLE
Schools

### PR DESCRIPTION
**Pull request by jrgrantham**
# Description

- School browse page that displays schools based on distance from buyer.
- When clicking on school card, user is redirected to School page where art is diplayed.
- Buyer can buy art by clicking on art card and entering card details.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change status

- [x] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

User steps on interface

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation, if relevant
- [x] I have run the app using my feature and ensured that no functionality is broken
- [x] My changes generate no new warnings
- [x] There are no merge conflicts

### CODE REVIEW

File: src/Components/BrowseComponents/BrowseArtContainer.js [hunk 1]
```diff
@@ -3,85 +3,100 @@ import { connect } from 'react-redux';
...
+    return null;
+  };
+  const { browseArtState } = props;
+  useEffect(() => {
+    axiosWithBase()
+      .get(
+        `/art/search?searchQuery=${browseArtState.searchQuery}&pagination=12&filter=${browseArtState.filter}&sortBy=${browseArtState.sortBy}&sortType=${browseArtState.sortType}&page=${page}`
+      )
+      .then(res => {
+        debugger;
```
Comment (iamcristos) [inline src/Components/BrowseComponents/BrowseArtContainer.js]: can you remove the debugger here

File: src/Components/BrowseComponents/BrowseCard.js [hunk 2]
```diff
@@ -12,6 +12,7 @@ function BrowseCard(props) {
         const test = props.browseArtState.art.filter(x => {
             return x.public_picture_id === props.id;
         })
+        debugger
```
Comment (iamcristos) [inline src/Components/BrowseComponents/BrowseCard.js]: also, remove the debugger here

File: src/Components/BuyModalComponents/DisplayArtModal.js [hunk 3]
```diff
@@ -11,6 +11,7 @@ function DisplayArt(props) {
     const distance = () => {
         const {userLocation} = props.loggedInUser;
         const {userId} = Object.keys(props.browseArtState.artInModal).length && props.browseArtState.artInModal.userId
+        debugger
```
Comment (iamcristos) [inline src/Components/BuyModalComponents/DisplayArtModal.js]: here also

File: src/Components/BrowseComponents/BrowseArtContainer.js (hunk 1 above)
Comment (petrussola) [reply src/Components/BrowseComponents/BrowseArtContainer.js]: done!

File: src/Components/BuyModalComponents/DisplayArtModal.js (hunk 3 above)
Comment (petrussola) [reply src/Components/BuyModalComponents/DisplayArtModal.js]: done!

File: src/Components/BrowseComponents/BrowseCard.js (hunk 2 above)
Comment (petrussola) [reply src/Components/BrowseComponents/BrowseCard.js]: done!

**Pull request merged**